streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -960,126 +960,65 @@ KTable<K, V> toTable(final Named named,
                          final Materialized<K, V, KeyValueStore<Bytes, byte[]>> materialized);
 
     /**
-     * Group the records of this {@code KStream} on a new key that is selected using the provided {@link KeyValueMapper}
-     * and default serializers and deserializers.
```

Comment:
Default serdes from the config are `null` now... Also, we should actually inherit form upstream first... Somewhat outdate. Removing it here.